### DISCUSSION TITLE
Error P1000 Can't reach database server

### DISCUSSION THREAD

**Discussion by NicolasChusseau** [prisma/prisma#19198] [Q&A] [answered]
Hello everyone,

I want to do a migration using prisma, supabase and redwood.
The problem is that when I try to do `yarn redwood prisma migrate dev` I get this : 
![image](https://github.com/prisma/prisma/assets/122353423/d96ef880-3af0-4d08-942c-82b3d9fcfad6)

My schema.prisma : 
```
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

generator client {
  provider      = "prisma-client-js"
  binaryTargets = "native"
}

model profiles {
  id         String   @id @default(uuid())
  updatedat  DateTime?
  firstName  String?
  lastName   String?
  email      String @unique
}
```
I have in my .env the DATABASE_URL found on supabase -> project settings -> database.
The value looks like this : `postgresql://postgres:password@db.oteaeccdiheyyajfvmqb.supabase.co:5432/postgres`

It actually works when I try my application on my localhost with `yarn redwood dev` command. I can read and write my profile table.

**Comment by nurul3101** [comment]
Hello @NicolasChusseau 👋 

Are you able to connect in a database IDE with this connection sting - `postgresql://postgres:password@db.oteaeccdiheyyajfvmqb.supabase.co:5432/postgres`?

Is your project open source, so that I can try to clone it and try to investigate the issue?

**Reply by NicolasChusseau** [reply, marked as answer]
I can connect to it using psql.   
I just tried again today and it just worked. I don't know why I didn't change anything.  
And sorry, the project is not open source and I can't modify it (I'm just an intern).

**Reply by nurul3101** [reply]
Thanks for getting back and letting us know. I am glad to hear that this is resolved now. 👍
